master/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml

```diff
@@ -49,6 +53,93 @@ data:
 
 ---
 
+# This manifest creates a Service, which will be backed by Calico's Typha daemon.
+# Typha sits in between Felix and the API server, reducing Calico's load on the API server.
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: calico-typha
+  namespace: kube-system
+  labels:
+    k8s-app: calico-typha
+spec:
+  ports:
+    - port: 5473
+      protocol: TCP
+      targetPort: calico-typha
```

Comment:
Do we use the name part of the port anywhere?

We could just drop that and the name on the pod's port as well if we're not using it - they're not required.

Reply:
I checked; Felix _does_ use this when it's doing its lookup.